Spigot-Server-Patches/0398-AreaEffectCloud-should-always-check-radius.patch

```diff
@@ -0,0 +1,43 @@
+From 3274a842dc3363b5f9d0369b80ec938b96a5f89e Mon Sep 17 00:00:00 2001
+From: William Blake Galbreath <blake.galbreath@gmail.com>
+Date: Mon, 27 May 2019 17:35:39 -0500
+Subject: [PATCH] AreaEffectCloud should always check radius
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityAreaEffectCloud.java b/src/main/java/net/minecraft/server/EntityAreaEffectCloud.java
+index 418c638d3..41d91a96d 100644
+--- a/src/main/java/net/minecraft/server/EntityAreaEffectCloud.java
++++ b/src/main/java/net/minecraft/server/EntityAreaEffectCloud.java
+@@ -163,6 +163,12 @@ public class EntityAreaEffectCloud extends Entity {
+         super.tick();
+         boolean flag = this.l();
+         float f = this.getRadius();
++        // Paper start
```

Comment:
Please open (or find an existing) a vanilla tricket if this is a vanilla bug, and then add it in a comment - `// Paper start - fix MC-??????`.